include/pagmo/batch_evaluators/member_bfe.hpp

```diff
@@ -0,0 +1,36 @@
+/* Copyright 2017-2018 PaGMO development team
+
+This file is part of the PaGMO library.
+
+The PaGMO library is free software; you can redistribute it and/or modify
+it under the terms of either:
+
+  * the GNU Lesser General Public License as published by the Free
+    Software Foundation; either version 3 of the License, or (at your
+    option) any later version.
+
+or
+
+  * the GNU General Public License as published by the Free Software
+    Foundation; either version 3 of the License, or (at your option) any
+    later version.
+
+or both in parallel, as here.
+
+The PaGMO library is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+for more details.
+
+You should have received copies of the GNU General Public License and the
+GNU Lesser General Public License along with the PaGMO library.  If not,
+see https://www.gnu.org/licenses/. */
+
+#ifndef PAGMO_BATCH_EVALUATORS_MEMBER_BFE_HPP
+#define PAGMO_BATCH_EVALUATORS_MEMBER_BFE_HPP
+
+// NOTE: the member_bfe is defined in bfe.hpp
```

Comment:
trust you here, but oh god reminds me of the hypervolume circular deps maddness ...

Reply:
Don't worry, this is all gone away in the library branch.